posts/2024-02-01-january-ecodev.mdx

```diff
@@ -0,0 +1,123 @@
+
+<!--truncate-->
+
+The EcoDev team at Waku covers community, growth, partnerships, and communications. Each month, we will share the key highlights and progress of the team. You can [subscribe](https://waku.org/subscribe/) to the monthly newsletter if you prefer to receive an abridged version of these updates in your inbox. 
+
+
+## Events and Hackathons
+
+In 2023, Waku participated in three hackathons and eight events across four continents. We are gearing up for another year full of events; expect to see us all around the globe once again. Waku will be attending two key in-person events in Q1 2024. We will focus on growing the developer community and meeting with projects to support further integrations with Waku.
+
+**ETHDenver 23rd February - 3rd March**
+
+Meet the Waku team attending ETHDenver: Aaron Bendesrky, Waku Program Manager at Logos, Pedro Lains, Business Development Lead, and Guru T, Developer Relationships Engineer. ETHDenver is a great opportunity to meet the Waku team face to face; atendees can book an appointment with the team [here](https://waku.org/events/eth-denver/).
+
+**ETHLatam 13th - 14th March**
+
+Waku has $3,500 in bounties on offer during the hackathon. The team on the ground will support the hackers across the two days in building their projects for submission, and we expect to see many more new use cases for Waku.
+
+Anybody hosting a side event at ETHDenver or ETHLatam and would like to collaborate with Waku, drop us message on [Discord](https://discord.com/invite/gMPAzmcDER). 
+
+
+## Press and Media
+
+**Vitalik Buterin Recognises Waku: **This month, Waku received an excellent shout-out from Vitalik Buterin, who recognised Waku as the successor of Whisper, the original peer-to-peer communications protocol for Ethereum. In a [blog post](https://vitalik.eth.limo/general/2023/12/28/cypherpunk.html#ethereum-as-part-of-a-broader-technological-vision), Buterin supported Waku as the realisation of the cypherpunk messaging protocol imagined by Gavin Wood.
+
+The story was also covered in [CoinTelegraph](https://cointelegraph.com/press-releases/vitalik-buterin-recognises-waku-as-the-peer-to-peer-communications-layer-of-gavin-woods-web3-vision), [CryptoDaily](https://cryptodaily.co.uk/2024/01/meet-waku-the-peer-to-peer-communications-protocol-recognized-by-vitalik-buterin), and [The Defiant](https://thedefiant.io/waku-the-p2p-communication-protocol-for-web3-recognized-by-vitalik-buterin). 
+
+**Waku was voted as a top privacy project of 2023**: Web3 Privacy Now included Waku on their long list of [top privacy-focused projects](https://mirror.xyz/0x0f1F3DAf416B74DB3DE55Eb4D7513a80F4841073/LdOt0mlG_nXqhnmLf7s-X13NDNdpwQw72ESKaJq-0zA) of the year. The finalists were voted on by over 100 privacy enthusiasts, developers, researchers, cryptographers, cypherpunks, and community builders from DarkFi, Nym, Swarm, Logos, Railgun, and many other organisations.
+
+
+## Technical Releases
```

Comment:
There was more technical reealse than that. Could we dig in the waku wednesday updates to find more relevant?
Many @chair28980 can help extract relevant update from weekly updates (and acutally point to roadmap website))